src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -4,74 +4,85 @@
 
 package frc.robot.subsystems;
 
-import edu.wpi.first.wpilibj2.command.Command;
-import edu.wpi.first.wpilibj2.command.SubsystemBase;
-import frc.robot.Constants;
-import edu.wpi.first.wpilibj.smartdashboard.*;
-
-import com.ctre.phoenix6.hardware.TalonFX;
-import com.ctre.phoenix6.controls.VelocityVoltage;
+import com.ctre.phoenix6.configs.FeedbackConfigs;
+import com.ctre.phoenix6.configs.MagnetSensorConfigs;
 import com.ctre.phoenix6.controls.MotionMagicVoltage;
+import com.ctre.phoenix6.hardware.CANcoder;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.ctre.phoenix6.signals.AbsoluteSensorRangeValue;
+import com.ctre.phoenix6.signals.FeedbackSensorSourceValue;
+import com.ctre.phoenix6.signals.SensorDirectionValue;
 
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import frc.robot.Constants;
+import frc.robot.generated.TunerConstants;
 
 public class Shooter extends SubsystemBase {
 
-  private final TalonFX pivotMotor = new TalonFX(14, Constants.OperatorConstants.canivoreSerial);
+  public final TalonFX pivotMotor = new TalonFX(14, Constants.OperatorConstants.canivoreSerial);
   private final TalonFX leftShootMotor = new TalonFX(5, Constants.OperatorConstants.canivoreSerial);
   private final TalonFX rightShootMotor = new TalonFX(13, Constants.OperatorConstants.canivoreSerial); 
   private double targetVelocity = 0;
   private double motorVelocity = 0;
   private boolean motorsAtShootingSpeed = false;
-  private VelocityVoltage velocity = new VelocityVoltage(targetVelocity);
-  private MotionMagicVoltage motionMagicVoltage = new MotionMagicVoltage(0.1);
+  private MotionMagicVoltage motionMagicVoltage = new MotionMagicVoltage(0);
```

Comment:
Have you tuned the required Kp, Kd, and Kg parameters yet for this wrist? It's fine if we tune them in Phoenix Tuner X, but I'd like to have them in code too. This is because, if we have to swap a motor out, we can just update it to have the right CAN id and our code will handle updating all of the values to have the correct K values.